docker-compose.example.yml

```diff
@@ -104,7 +98,7 @@ services:
   klink:
     image: "docker.klink.asia/images/k-link:0.1.0"
     ports:
-      - 8080:80
+      - 8181:80
```

Comment:
Why this port was changed? I would use 8080 or 8000 as default. If someone installs other things that listen on that port is not our problem

Reply:
Because I want to have the out-of-the-box K-Box being able to potentially talk to an out-of-the-box K-Link. To avoid them having the same port.

Reply:
ok, it was just a weird port number

Reply:
I would have selected 8000, but is not a problem that blocks your pull request